data/fwupd.conf.md

```diff
@@ -0,0 +1,238 @@
+% fwupd.conf(5) @PACKAGE_VERSION@ | Configuration File Formats
+
+NAME
+====
+
+**fwupd.conf** — configuration file for the fwupd daemon.
+
+SYNOPSIS
+========
+
+The `/etc/fwupd/fwupd.conf` file is the main configuration file for the fwupd daemon.
+The complete description of the file format and possible parameters are documented here for reference purposes.
+
+FILE FORMAT
+===========
+
+The file consists of a multiple sections with optional parameters. Parameters are of the form:
+
+    [section]
+    key = value
+
+The file is line-based, each newline-terminated line represents either a comment, a section name or a parameter.
+
+Section and parameter names are case sensitive.
+
+Only the first equals sign in a parameter is significant.
+Whitespace before or after the first equals sign is discarded as is leading and trailing whitespace in a parameter value.
+Internal whitespace within a parameter value is retained.
+
+Any line beginning with a hash (`#`) character is ignored, as are lines containing only whitespace.
+
+The values following the equals sign in parameters are all either a string (no quotes needed) or a boolean, which may be given as **true** or **false**.
+Case is not significant in boolean values, but is preserved in string values. Some items are also numeric.
+
+DAEMON PARAMETERS
+=================
+
+The `[daemon]` section can contain the following parameters.
+
+**DisabledDevices=**[GUID];[GUID]
+
+    Allow blocking specific devices by their GUID, using semicolons as delimiter.
+
+**DisabledPlugins=**[PLUGIN_NAME];[PLUGIN_NAME]
+
+    Allow blocking specific plugins by name.
+    Use **sudo fwupdtool get-plugins** to get the list of plugins.
```

Comment:
I don't think we want to be telling people to use fwupdtool do we?

Reply:
Fixed.